classes/person.rb

```diff
@@ -4,14 +4,15 @@
 class Person < Nameable
   # accessor get methods
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ]  Great job for creating Person class, but seemed that you might have forgotten to add `add_rental method`. This method should present in your Person class as well to help you to add both book and date when a person rented, for more [check ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship`)